packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx

```diff
@@ -17,21 +17,15 @@ export interface ActiveVisit {
 
 export function useActiveVisits() {
   const { data: currentUserSession } = useCurrentSession();
-  const sessionLocation = currentUserSession?.data?.sessionLocation?.uuid;
   const startDate = dayjs().format('YYYY-MM-DD');
-
+  const sessionLocation = currentUserSession?.data?.sessionLocation?.uuid;
   const customRepresentation =
     'custom:(uuid,patient:(uuid,identifiers:(identifier,uuid),person:(age,display,gender,uuid)),' +
     'visitType:(uuid,name,display),location:(uuid,name,display),startDatetime,' +
-    'stopDatetime)&fromStartDate=' +
-    startDate +
-    '&location=' +
-    sessionLocation;
-
-  const { data, error, isValidating } = useSWR<{ data: { results: Array<Visit> } }, Error>(
-    `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}`,
-    openmrsFetch,
-  );
+    'stopDatetime)&fromStartDate=';
+  const url =
+    `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}` + startDate + '&location=' + sessionLocation;
```

Comment:
I find the previous version more readable. It seems strange to say `fromStartDate=` and then leave it hanging.